std/assembly/util/number.ts

```diff
@@ -374,23 +374,23 @@ export function itoa32(value: i32, radix: i32): String {
   }
   if (!value) return "0";
 
-  var sign = value >>> 31;
+  var sign = (value >>> 31) << 1;
```

Comment:
This feels a little odd. Does it really add much to use `2` here?

Reply:
it's just optimization. Perhaps make sense call this `sign2` or `signAndMul`

Reply:
Wondering about it since previously there were 4 uses of `sign` and now there are 8, plus that it is harder to grasp now.

Reply:
`sign` uses more and this basically should be faster due to adding sign skip one unnecessary iteration of main core process

Reply:
Perhaps I don't properly understand this. Can you explain how one iteration is removed?

Reply:
Before this PR we didn't add sign value (1) for `buffer`'s offset but still changing the sign (`value = -value`) for negatives, so `-32768` for example required 6 iterations in code process. However after change sign it becomes to `32768` required only 5 iterations. Now this fixed and for negative values we use exactly as many iterations as necessary.

Reply:
In other words in methods like `utoa32_dec_core` the number of iterations depended on the presence or absence of a sign. That's not the case now.